### PR TITLE
Refactor vLLM guide

### PR DESCRIPTION
**Pull request by dyastremsky**
- Use uncompressed conda environment to reduce time to conda-pack and to reduce need for Triton to unpack (from Ryan McCormick's testing, this shrinks the model load time from 170 seconds to 13 seconds).
- Remove unnecessary Nsight Systems files from conda environment
- Use one container for steps 1 and 2 for simplicity (script will now check if step 1 is complete and exit if so)

Tested locally.

If step 1 is re-run:
image

After following steps:
image
image

### CODE REVIEW

File: Quick_Deploy/vLLM/README.md [hunk 1]
```diff
@@ -92,21 +107,22 @@ I0901 23:39:08.772522 1 http_server.cc:187] Started Metrics Service at 0.0.0.0:8
...
 
-Within the container, run the [`client.py`](client.py) as:
+Within the container, run [`client.py`](client.py) with:
 
 ```
 python3 client.py
 ```
 
-The client reads prompts from [prompts.txt](prompts.txt) file, sends them to Triton server for inference and stores the results into a file named `results.txt` by default.
+The client reads prompts from [prompts.txt](prompts.txt) file, sends them to Triton server for
```
Comment (rmccorm4) [inline Quick_Deploy/vLLM/README.md:124]:
```suggestion
The client reads prompts from the [prompts.txt](prompts.txt) file, sends them to Triton server for
```

File: Quick_Deploy/vLLM/README.md [hunk 2]
```diff
@@ -116,9 +132,12 @@ Storing results into `results.txt`...
 PASS: vLLM example
 ```
 
-You can inspect the contents of the `results.txt` for the response from the server. `--iterations` flag can be used with the client to increase the load on the server by looping through the list of provided prompts in [`prompts.txt`](prompts.txt).
+You can inspect the contents of the `results.txt` for the response from the server. `--iterations`
```
Comment (rmccorm4) [inline Quick_Deploy/vLLM/README.md:135]:
```suggestion
You can inspect the contents of the `results.txt` for the response from the server. The `--iterations`
```

File: Quick_Deploy/vLLM/README.md [hunk 3]
```diff
@@ -116,9 +132,12 @@ Storing results into `results.txt`...
...
+flag can be used with the client to increase the load on the server by looping through the list of
+provided prompts in [`prompts.txt`](prompts.txt).
 
-When you run the client in verbose mode - with `--verbose` flag, the client will print more details about the request/response transactions.
+When you run the client in verbose mode - with `--verbose` flag, the client will print more details
```
Comment (rmccorm4) [inline Quick_Deploy/vLLM/README.md:139]:
```suggestion
When you run the client in verbose mode with the `--verbose` flag, the client will print more details
```

File: Quick_Deploy/vLLM/gen_vllm_env.sh [hunk 4]
```diff
@@ -34,15 +34,29 @@
...
+
+# If this script runs, clean up previous targets.
+rm -rf $ENV_DIR $STUB_FILE
+
 # Install and setup conda environment
-file_name="Miniconda3-latest-Linux-x86_64.sh"
-rm -rf ./miniconda  $file_name
-wget https://repo.anaconda.com/miniconda/$file_name
+FILE_NAME="Miniconda3-latest-Linux-x86_64.sh"
+rm -rf ./miniconda  $FILE_NAME
```
Comment (rmccorm4) [inline Quick_Deploy/vLLM/gen_vllm_env.sh:53]:
```suggestion
rm -rf ./miniconda $FILE_NAME
```

**Comment by rmccorm4** [comment]
Nice improvements! I'm seeing ~16mins for `./gen_llm_env.sh` which is OK, but I'm looking into a quick idea on the side,

Otherwise, after env is generated I'm seeing ~30 seconds for first serving of model repository (to download opt model), and ~12 seconds for subsequent server startups after model has already been downloaded. This is a significant improvement to UX!

**Review by rmccorm4 — APPROVED** [review]

**Comment by dyastremsky** [comment]
@tanmayv25 Would you be able to merge this? I will create a separate PR for moving to pip, if successful. I would like to keep these changes modular to make it easier to use the Conda version in the future, if necessary.

**Review by tanmayv25 — APPROVED** [review]

**Pull request merged**